### ISSUE TITLE
ID of new card created via newCardTemplate component being overridden when passed back to parent

### ISSUE THREAD

**Issue by CodyFitzpatrick** [rcdexta/react-trello#117]
Simply put, I am making use of a custom NewCard template, e.g., `<NewCard />`.

Within that component, in the `handleAdd` method, I am performing some actions including pinging an API to get an accurate ID for the new card.

When I pass the card's ID along with other pertinent information to the parent component (like text of card, position, and some other attributes), **everything gets through as expected except for the ID (see attached console screenshot)**. 

**It is as though the ID is being hashed, obfuscated, or straight-up overwritten on its way up to the parent handler.**

Any ideas on how to fix this? I want to pass a valid card ID in order to protect other logic (like sorting of cards within lanes).

![screen shot 2018-10-12 at 8 32 52 pm minified](https://user-images.githubusercontent.com/5438023/46899206-7a5d9080-ce5e-11e8-81d5-51ed1fc7cec4.png)

Here is the logic within the `handleAdd` method within my custom `NewCard` component producing the output pictured above. I have tried passing the ID in multiple ways, but unfortunately see the same result each time.

<img width="610" alt="screen shot 2018-10-12 at 8 41 56 pm" src="https://user-images.githubusercontent.com/5438023/46899282-50589e00-ce5f-11e8-8c8f-d481cf777c2d.png">

**Comment by CodyFitzpatrick** [comment]
Per #119, it looks like new card IDs are being overwritten in react-trello/src/components/Lane.js with the ID generator function, uuidv1(). Here's a snippet of the culprit:
```
addNewCard = params => {
    const laneId = this.props.id
    const id = uuidv1()
    this.hideEditableCard()
    let card = {...params, id}
    this.props.actions.addCard({laneId, card})
    this.props.onCardAdd(card, laneId)
}
```

**Comment by rcdexta** [comment]
Fix available in 2.0.3
